service/reducers/entities/helpers.js

```diff
@@ -85,15 +69,117 @@ export function addPosts(state, action) {
         return 0;
     });
 
-    newState.postsInfo = {
-        ...newState.postsInfo,
-        [id]: {
-            ...newState.postsInfo[id],
-            postList: combinedPosts
-        }
+    return {
+        ...state,
+        [channelId]: combinedPosts
+    };
+}
+
+function storePost(state, action) {
+    const post = action.data;
+    const channelId = post.channel_id;
+
+    const postInfo = state[channelId];
+    const postList = postInfo ? postInfo.postList : {};
```

Comment:
Why does `storePosts` basically have `postList = state[channelId]`, but this has `postList = state[channelId].postList`? Shouldn't `state[channelId]` be the same thing for both of these?